Automattic/wp-calypso
# Domains: Hide auto-renew toggle for bundled registered domains

### Changes proposed in this Pull Request

This PR hides the auto-renew toggle for registered domains which are part of a bundle, i.e. which are renewed along with their plan. This should fix issue #60366.

### Testing instructions

- Build this branch locally or open the live Calypso link
- Select a site where you have a bundled registered domain
- Go to Upgrades > Domains
- Select the bundled domain
- Ensure that the auto-renew toggle is not shown
